sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -502,4 +505,24 @@ void from(CalcitePlanner planner)
                                          + " to " + this);
     }
   }
+
+  /** Define some tool members and methods for hints. */
+  private static class HintTools
+  {
+    static final HintStrategyTable HINT_STRATEGY_TABLE = createHintStrategies();
+
+    /**
+     * Creates hint strategies.
+     *
+     * @return HintStrategyTable instance
+     */
+    private static HintStrategyTable createHintStrategies()
```

Comment:
note: I'm not sure if the `HintTools` class is needed; you could just have this method alone...its already `static`

Reply:
We might need this as we would also have to add the predicates